src/codegen.rs

```diff
@@ -222,7 +233,7 @@ impl GerberCode for Aperture {
             Aperture::Rectangle(ref rectangular) => format!("R,{}", try!(rectangular.to_code())),
             Aperture::Obround(ref rectangular) => format!("O,{}", try!(rectangular.to_code())),
             Aperture::Polygon(ref polygon) => format!("P,{}", try!(polygon.to_code())),
-            Aperture::Other(ref string) => panic!("not yet implemented"),
```

Comment:
Not really about this PR, but you do know about the [`unimplemented!`](https://doc.rust-lang.org/std/macro.unimplemented.html) macro?